parent/pom.xml

```diff
@@ -41,7 +41,7 @@
         <project.identifier>moditect</project.identifier>
         <project.repository>moditect/moditect</project.repository>
         <local.repository.path>/tmp/repository</local.repository.path>
-        <java.version>1.8</java.version>
+        <java.version>11</java.version>
```

Comment:
Is there a strong reason for doing that? One of the original motivations for ModiTect was to provide the ability to create module descriptors for a project, while still being on Java 1.8 for that project's own build. Naturally, this becomes less of a "feature" over time as folks upgrade, but if the update isn't strictly needed, then I'd wait a bit further (note most features of ModiTect require 9+ already anyways).

Reply:
I don't remember if Java 8 -> 11 upgrade was strictly necessary, but ASM upgrade definitely was, as ASM 6 was failing to process Java 11 byte code because it didn't understand class version.

That said, Moditect imo only makes sense with Java 9+, as that's where JPMS was added, and 11 is the first LTS release since then. I also don't think compiling Moditect itself using Java 11 forces projects using it to use 11 as well. Finally, Java 8 is getting a bit old in the tooth, with 17 already being out for almost a year, and 21 only a year away, so I'm not sure Java 8 support is even relevant at this point.